itou/templates/news/news.html

```diff
@@ -0,0 +1,64 @@
+            <ul class="list-group list-group-collapse list-group-flush" id="news-list">
+                {% for month_news in news_page %}
+                    <li class="list-group-item list-group-item-action">
+                        <button class="d-block w-100"
+                                data-bs-toggle="collapse"
+                                data-bs-target="#news-{{ month_news.id }}"
+                                type="button"
+                                aria-expanded="{% if forloop.counter > 1 %}false{% else %}true{% endif %}"
+                                aria-controls="news-{{ month_news.id }}">
+                            <div class="h3 mb-0">
+                                {{ month_news.start_date|date:"F Y"|capfirst }}
+                                <span class="badge badge-sm bg-emploi-light text-info rounded-pill ms-2" aria-label="nombre d'articles">{{ month_news.count_items }}</span>
+                            </div>
+                        </button>
+                        <div class="mt-3{% if forloop.counter > 1 %} collapse{% else %} collapse show{% endif %}" id="news-{{ month_news.id }}" aria-controls="news-{{ month_news.id }}">
+                            {% for news_item in month_news.items.all %}
+                                <div class="row mb-3">
+                                    <div class="col-12 col-md-4{% if news_item.image %} mb-3{% else %} d-none d-md-inline{% endif %}">
+                                        {% if news_item.image %}
+                                            <img src="{{ news_item.image.url }}"
+                                                 loading="lazy"
+                                                 alt="{{ news_item.image_alt_text|default:'' }}"
+                                                 {% if not news_item.image_alt_text %}aria-hidden="true"{% endif %}
+                                                 class="img-fluid img-fitcover img-thumbnail" />
+                                        {% else %}
+                                            <img src="{% static_theme_images 'ico-bicro-important.svg' %}" loading="lazy" alt="" class="img-fluid img-fitcover img-thumbnail img-muted" aria-hidden="true" />
+                                        {% endif %}
+                                    </div>
+                                    <div class="col-12 col-md-8">
+                                        <div aria-label="groupes d'utilisateurs concernés">
+                                            {% for tag in news_item.user_kind_labels %}
+                                                <span class="tag tag-base bg-info-lighter text-info">{{ tag|upper }}</span>
+                                            {% endfor %}
+                                        </div>
+                                        <p class="h4 my-2">{{ news_item.title }}</p>
+                                        <p>{{ news_item.description }}</p>
```

Comment:
Le fait de respecter les retours à la lignes originaux devrait éviter pas mal de mauvaises surprises et permettre un peu plus de flexibilité sur la mise en page pour les personnes remplissant le changelog.
```suggestion
                                        <p>{{ news_item.description|linebreaksbr }}</p>
```